data/remnant/remnant missions.txt

```diff
@@ -4479,3 +4479,140 @@ mission "Remnant: Expanded Horizons Astral job"
 		payment 500000
 		"reputation: Remnant" ++
 		dialog "When you land, a researcher quickly retrieves the data crystals with the information about the surveyed system and hands you the logistics allotment of <payment>."
+
+
+
+outfit "_Ion Storm Timer: Generator"
+	# This generator will fail when an ion storm comes and the ship runs out of energy
+	"hull repair rate" 60000
+	"hull energy" 1
+
+
+
+ship "_Ion Timer Ship"
+	# The Timer Weapon causes this ship to continually lose hull points
+	# The _Ion Storm Timer: Generator outfit counteracts this until there is an ion storm
+	# sprite "ship/shuttle" # uncomment this for debugging
+	attributes
+		"heat dissipation" 1
+		"hull" 1.0
+		"mass" 100
+		"drag" 5
+		"automaton" 1
+		"energy generation" 1
+		"thrust" 10
+		"turn" 600
+	outfits
+		"Timer Weapon"
+		"_Ion Storm Timer: Generator"
+	gun 0 0
+
+
+
+mission "Remnant: Expanded Horizons Storms 1"
+	name "Ember Waste Storm Scanning"
+	description "Dawn has asked you to go scan one of the Ember Waste's storms."
+	passengers 1
+	source
+		government "Remnant"
+	waypoint
+		system "Aescolanus"
+	to offer
+		has "Remnant: Expanded Horizons Astral 2: done"
+		random < 30
+	on offer
+		require "Research Laboratory"
+		conversation
+			`As you cross the starport you are suddenly ambushed by Dawn appearing out of a side hallway. "Captain <first>! I have been hoping to catch you. I need some help." She hurries up, waving a datapad full of charts. "I have some additional leads for my research project, and I need a pilot who can take me out to Aescolanus." She looks at you hopefully.`
+			choice
+				`	"Sure, I am always willing to help."`
+				`	"Not today, sorry."`
+					decline
+			`	"Excellent! You already have a research laboratory onboard, and it has all the equipment I need. Now, before we head out, you should know that the goal of this mission is to do some analysis on the leading edge of a storm that will be breaking on the system shortly. We will not have to stay for the whole storm, but we will have to be there for at least the first part."`
```

Comment:
I don't think it's necessary, but you could add a branch here in case the player doesn't have a research laboratory that defers the mission and remove the requirement up top. That way, the player knows what they need to do to continue the chain (assuming there will be more after this).
Again, not necessary for this PR, so you could probably add it in later if you wanted.

Reply:
For the moment, nothing depends on this mission; and no major plot lines are planned to depend on it either, although it is likely to be a trigger for a branch in one or two spots.  Given that Research Laboratories are actually required for the main campaign, I think there's at least a decent chance that the player will get this before they get rid of the lab again.

Likewise, it is my hope that the Research Lab's other abilities are sufficiently useful to not be perceived as "Dead-weight," and thus the player keeps it around and can get the mission.

I might revisit this down the road after the general player base has had a chance to play it. Depending on how things turn out, whether or not this mission needs a branch is something to be decided at a later date after more people have had the chance to play it.

Alternatively, we might want to consider buffing the Research Lab. For instance, giving it a decent chunk of tactical scanning (say, at about half the value of the Salvage Scanner) would probably be a significant perk for many people), and would be thematically appropriate. Even better would be if we could give it a multiplicative value. For instance, having the Research lab apply a x2 modifier to the sum of the tactical scan value of your ship. Basically, your science officer can coax better performance out of whatever scanning you have. But they can't help if you don't already have said scanning capability.